ios/COVIDSafePaths/AppDelegate.m

```diff
@@ -80,6 +80,16 @@ - (void)application:(UIApplication *)application didReceiveLocalNotification:(UI
 }
 
 - (void)applicationWillTerminate:(UIApplication *)application {
+    UILocalNotification *_localNotification = [[UILocalNotification alloc] init];
+    _localNotification.fireDate = [NSDate dateWithTimeIntervalSinceNow:5];
+    _localNotification.timeZone = [NSTimeZone defaultTimeZone];
+    _localNotification.alertTitle = @"Private Kit Enabled";
+    _localNotification.alertBody = @"Private Kit is securely storing your GPS coordinates once every five minutes on this device.";
```

Comment:
Use the app name: COVID Safe Paths is securely...

Reply:
Sure I will update